ktor-client/ktor-client-plugins/ktor-client-auth/common/src/io/ktor/client/plugins/auth/Auth.kt

```diff
@@ -40,6 +42,7 @@ public val AuthCircuitBreaker: AttributeKey<Unit> = AttributeKey("auth-request")
  * You can learn more from [Authentication and authorization](https://ktor.io/docs/auth.html).
  *
  * [providers] - list of auth providers to use.
+ * [isUnauthorized] - lambda function to control whether a response is unauthorized and should trigger a re-auth.
```

Comment:
Something else: this revers to variables that are actually defined on AuthConfig. Shouldn't the docstring be defined there instead or at least refer to the full path `[AuthConfig.providers]`? I just copied the existing code, but maybe this should be fixed, too.